mne_bids/dig.py

```diff
@@ -225,7 +226,8 @@ def _coordsystem_json(raw, unit, orient, coordsystem_name, fname,
     fname : str
         Filename to save the coordsystem.json to.
     kind : str
-        Type of the data as in ALLOWED_KINDS.
+        Type of the data. One of ``'meg'``, ``'eeg'``, ``'ieeg'``.
```

Comment:
I would avoid the double ticks

Reply:
But they make the rendered docs easier to read :)

Reply:
but it's a private function. Anyway, nitpick :)

Reply:
> but it's a private function.

goot point. Let's drop the backticks then :)